gigamon/assets/logs/gigamon.yaml

```diff
@@ -0,0 +1,102 @@
+id: gigamon
+backend_only: true
```

Comment:
```suggestion
backend_only: false
```

Reply:
@emarsha94 , not sure, it is asking to map as "true" during a run. Please confirm whether i need to change it as false.